### ISSUE TITLE
chainable methods

### ISSUE THREAD

**Issue by jywarren** [publiclab/image-sequencer#40]
if each method `loadImages()`, `addStep()` etc returns the sequencer itself, we can chain, like:

```js
sequencer.loadImages(image_name, image_src)
         .addStep(...)
         .run();
```

That's also a really succinct way to demonstrate the entire usage cycle of the library, and would be great to include in the README!

**Comment by ccpandhare** [comment]
This is actually a great idea! Will add this to the project.

**Comment by ccpandhare** [comment]
Fixed in #44.
